roles/container-engine/docker/tasks/main.yml

```diff
@@ -127,6 +127,7 @@
   when: ansible_pkg_mgr == 'apt'
   changed_when: false
   with_items:
+    - containerd.io
```

Comment:
don't forget to remove the hold first (ligne 95), right now we are removing the hold for `containerd` not `containerd.io`

Reply:
Should we remove both `containerd` and `containerd.io` ? It seems the `containerd.io` is a recent thing with `docker-ce` and `containerd` is the one from the ubuntu official repos. My understanding is that kubespray prefers the docker-ce so it would make sense to move the logic to `containerd.io`.

Reply:
In the latest push I modified the line 95 from `containerd` to `containerd.io` since that is the proper name of the docker.com package

Reply:
See https://github.com/kubernetes-sigs/kubespray/issues/7276, @drivard suggest using `"{{ containerd_package }}"`

Reply:
Indeed that looks like a much better approach, I took that approach int the latest change.